src/App.js

```diff
@@ -1,7 +1,35 @@
 import './App.css';
-
+import Home from './views/Home/Home';
+import { Switch } from 'react-router-dom';
+import { React } from 'react-router-dom';
+import { BrowserRouter } from 'react-router-dom';
+import { Route } from 'react-router-dom';
+import Header from './components/Header';
+import Characters from './views/Characters/Characters';
+import Films from './views/Films/Films';
+import Books from './views/Books/Books';
 function App() {
-  return <div className="App"></div>;
+  return (
+    <BrowserRouter>
+      <Header />
+      <div className="App">
+        <Switch>
+          <Route exact path="/">
+            <Home />
+          </Route>
+          <Route exact path="/characters">
+            <Characters />
+          </Route>
+          <Route exact path="/films">
+            <Films />
+          </Route>
+          <Route exact path="/books">
+            <Books />
+          </Route>
```

Comment:
Great usage of 'exact path' so that our routes are not confused amongst similar routes.